src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.CURLcode.cs

```diff
@@ -12,6 +12,7 @@ internal enum CURLcode
             CURLE_UNSUPPORTED_PROTOCOL  =  1,
             CURLE_NOT_BUILT_IN = 4,
             CURLE_COULDNT_RESOLVE_HOST  =  6,
+            CURLE_ABORTED_BY_CALLBACK = 42,
```

Comment:
The native shim needs to be updated with this value, both adding it to the PAL enums as well as adding static asserts to verify it matches the value we expect.